CHANGELOG.md

```diff
@@ -0,0 +1,168 @@
+# Changelog
+All notable changes to the SpatialOS Game Development Kit for Unreal will be documented in this file.
+
+The format of this Changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] - 2019-xx-xx
```

Comment:
Should we preemptively add the `Features`, `Bug fixes` and `Known Issues` sections so we have a place to put the new ones.

Reply:
Ignore me